libtenzir/src/diagnostics.cpp

```diff
@@ -27,7 +28,7 @@ namespace {
 void trim_and_truncate(std::string& str) {
   using namespace std::string_view_literals;
   boost::trim(str);
-  if (str.size() > 2000) {
+  if (str.size() > 100000) {
```

Comment:
@dominiklohmann We are okay with this, right? I added some truncation in `json.cpp` where it is missing, should be fine hopefully. Putting this on auto-merge now, feel free to resolve.